internal/store/hpa.go

```diff
@@ -132,7 +132,7 @@ var (
 
 					for j, m := range metrics {
 						metric := m
-						metric.LabelKeys = []string{"condition", "status"}
```

Comment:
labels are intended to be lexicographic-ally sorted, let's instead change the order these are passed in

Reply:
Sorted.